### ISSUE TITLE
Distorted Featured Images on Pages with Blog Posts block

### ISSUE THREAD

**Issue by nickpagz** [Automattic/themes#3517]
## Steps to replicate

Add a blog posts block to a page on a site with the Twenty Twenty-One theme active.
Set it to show featured images and titles and excerpts.

## Result

The featured images appear on the live site, and they are distorted (tall and narrow instead of landscape).

## Expected

The blog posts block to display similarly to how it looks in the editor.

The CSS code from [this GH issue](https://github.com/Automattic/themes/issues/2525) worked to fix it:

```
/* fix images styles of blog posts block  | #-hc/zd bk */
body.page .hentry.has-post-thumbnail .wpnbha .wp-post-image, 
body.single .hentry.has-post-thumbnail .wpnbha .wp-post-image {
  position: static;
  min-height: auto;
  height: auto !important;
  transform: none;
}
/* fix spacing of blog posts block  | #-hc/zd bk */
body.single:not(.title-with-content) .hentry.has-post-thumbnail:not(.format-video) .wpnbha .entry-title,
body.single:not(.title-with-content) .hentry.has-post-thumbnail:not(.format-video) .wpnbha .entry-meta,
.wpnbha .entry-title a,
.wpnbha .entry-meta a {
  padding-left: 0;
  padding-right: 0;
}
```

* User report: #28584796-hc

**Comment by danieldudzic** [comment]
I'm seeing issues when setting the Blog Posts block to Wide or Full-width.

![Notification_Center](https://user-images.githubusercontent.com/905781/120218938-be357c00-c23a-11eb-9fb1-b828e9262348.png)

The above CSS fix doesn't resolve the issue for me, as it seems to be related to the incorrect cropping of the image coming from the block: https://x.files.wordpress.com/2021/01/visit_opening_hours_1.jpg?w=800&h=600&crop=1

Would you mind having a look, @MaggieCabrera ?

I'd be keen to learn your perspective on this.

p.s.
Could this potentially be related to: https://github.com/Automattic/newspack-blocks/issues/711 ?

**Comment by MaggieCabrera** [comment]
I have tried this with very vertical and very horizontal images and they all get cropped to the same ratio. This seems to be the intended effect from the block but it causes some images to expand too much if their original ratio is too different from the cropped one, even more so on full or wide width where the image will be stretched even further. Having an option to change the ratio would help with this, but this is definitely not a theme issue. I have tested TT1, Spearhead, and some Varia children too, they all behave the same.
